packages/protocol/test/L1/TaikoL1TestBase.sol

```diff
@@ -68,15 +70,38 @@ abstract contract TaikoL1TestBase is TestBase {
         pv = new PseZkVerifier();
         pv.init(address(addressManager));
 
+        sv = new SgxVerifier();
+        sv.init(address(addressManager));
+        address[] memory initSgxInstances = new address[](2);
+        initSgxInstances[0] = SGX_X_0;
+        initSgxInstances[1] = SGX_VARIANT;
+        sv.addInstances(initSgxInstances);
+
+        sgxZkVerifier = new SgxAndZkVerifier();
+        sgxZkVerifier.init(address(addressManager));
+
         gv = new GuardianVerifier();
         gv.init(address(addressManager));
 
-        cp = new OptimisticRollupConfigProvider();
+        gp = new GuardianProver();
+        gp.init(address(addressManager));
+        address[5] memory initMultiSig;
+        initMultiSig[0] = David;
```

Comment:
Test test specific code should go to their own class, the base class is suppose to be clean and lean.

Reply:
I added two TODOs in the test files, you can take care of them later and go ahead to merge this PR.